src/CosmosEditorManager.ts

```diff
@@ -76,7 +76,11 @@ export class CosmosEditorManager {
     }
 
     private async updateEditor(data: {}, textEditor: vscode.TextEditor): Promise<void> {
-        await util.writeToEditor(textEditor, JSON.stringify(data, null, 2));
+        if (typeof data === "string") {
+            await util.writeToEditor(textEditor, <string>data);
+        } else {
+            await util.writeToEditor(textEditor, JSON.stringify(data, null, 2));
```

Comment:
magic number

Reply:
I.e. extract const.  Are we using this elsewhere?  They should all use the same value

Reply:
It's used only in two places in the code base. Totally three possibly ambiguous use cases. 
Have an issue https://github.com/Microsoft/vscode-cosmosdb/issues/537, will track it there.